From 88f3d2dd31fbef7e539b2523724221e8e8e5a9f0 Mon Sep 17 00:00:00 2001
From: Falk <falkartis@gmail.com>
Date: Thu, 27 Oct 2016 18:10:48 +0200
Subject: [PATCH] allow to type just a instead of all

Typing all many times can be boring/annoying, So I propose to allow people to type just a for all.
(sometimes there can be very large duplicate sets)
---
 fdupes.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fdupes.c b/fdupes.c
index 7eee165..7a8a6ed 100644
--- a/fdupes.c
+++ b/fdupes.c
@@ -886,7 +886,7 @@ void deletefiles(file_t *files, int prompt, FILE *tty)
 	token = strtok(preservestr, " ,\n");
 	
 	while (token != NULL) {
-	  if (strcasecmp(token, "all") == 0)
+	  if (strcasecmp(token, "all") == 0 || strcasecmp(token, "a") == 0)
 	    for (x = 0; x <= counter; x++) preserve[x] = 1;
 	  
 	  number = 0;
